Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -117,26 +92,16 @@ public OrderTicket Sell(Symbol symbol, float quantity)
         /// <returns>Int Order Id.</returns>
         public OrderTicket Sell(Symbol symbol, decimal quantity)
         {
-            return Order(symbol, Math.Abs(quantity) * -1);
-        }
-
-        /// <summary>
-        /// Issue an order/trade for asset: Alias wrapper for Order(string, int);
-        /// </summary>
-        /// <seealso cref="Order(Symbol, decimal)"/>
-        public OrderTicket Order(Symbol symbol, double quantity)
```

Comment:
Changes to API can break thousands of algorithms, ideally we don't remove these unless absolutely neccessary